runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -1426,4 +1433,8 @@
 	 * @param speed speed
 	 */
 	void setOculusOrbNormalSpeed(int speed);
+
+	void openWorldHopper();
+
+	void hopToWorld(int world);
```

Comment:
Documentation

Reply:
Also, to prevent hopping to non-existant world, this could maybe take `World` instead. Everywhere we use it, we have `World` at hand already